lib/formatters/json.js

```diff
@@ -1,17 +1,19 @@
 import Linter from '../linter.js';
 
-export default class JsonPrinter {
+export default class JsonFormatter {
+  outputFileType = 'json';
```

Comment:
Is this used?

Reply:
I see, it is used in the bin script to tell `writeOutputFile` what extension to use by default. 

Should we name it something different to make it clear that it's just a default extension? (if the user provides a file name, this value is not used at all)